CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 * Fixes checksums for Ruby 2.6.4 [\#4769](https://github.com/rvm/rvm/pull/4769), 2.4.7 and 2.5.6 [\#4771](https://github.com/rvm/rvm/pull/4771)
 * Update TruffleRuby dependencies [\#4815](https://github.com/rvm/rvm/pull/4815)
 * Use ruby.git master instead of trunk [\#4840](https://github.com/rvm/rvm/pull/4840)
+* Fix RVM version check when using a version newer than the latest release [#4872](https://github.com/rvm/rvm/pull/4872)
```

Comment:
I was planning to do this fix ages ago and never found time! Thanks!

Reply:
:smiley: Could you merge the PR? Or should we wait for @mpapis' review?

Reply:
Spoke with @mpapis and he liked your PR, so I will merge it now :)